addons/website_sale_delivery/views/website_sale_delivery_view.xml

```diff
@@ -7,7 +7,7 @@
         <field name="model">delivery.carrier</field>
         <field name="inherit_id" ref="delivery.view_delivery_carrier_form"/>
         <field name="arch" type="xml">
-            <xpath expr="//div[@class='oe_title']" position="after">
+            <xpath expr="//div[@class='oe_title']" position="before">
```

Comment:
When you use a xpath with a class, it's better to use : 
          //div[contains(@class, your_class)] 
instead of
         //div[@class=your_class]
in case of there is another class added later on this div.